dapr/proto/runtime/v1/dapr.proto

```diff
@@ -194,12 +194,6 @@ service Dapr {
 
   // Delete a job
   rpc DeleteJob(DeleteJobRequest) returns (google.protobuf.Empty) {}
-
-  // Get a job
-  rpc GetJob(GetJobRequest) returns (GetJobResponse) {}
```

Comment:
We won't support these APIs until there is a use case.

Reply:
Please do not remove these APIs. They are invaluable to applications for discovery and "ensure" type operations. They are also part of the original proposal https://github.com/dapr/proposals/pull/44
I don't see any need to be removing these.

Reply:
Dapr does not offer this scenario. Also, the Get job is not as simple in a distributed scheduler since a job is not being executed on every instance. Reading from etcd does not mean the job is being executed either. We don't offer GetJob in reminders, for example.

Reply:
I will add Get but List needs a scenario to understand what we are solving for. I know we did not solve for observability yet.

Reply:
Added Get back.